src/Cake.Common/Tools/DotNetCore/Run/DotNetCoreRunner.cs

```diff
@@ -91,6 +91,10 @@ private ProcessArgumentBuilder GetArguments(string project, ProcessArgumentBuild
                 builder.Append(settings.Runtime);
             }
 
+            // Roll Forward Policy
+            builder.Append("--roll-forward");
+            builder.Append(settings.RollForward.ToString());
+
```

Comment:
```suggestion
            if (!(settings.RollForward is null))
            {
                // Roll Forward Policy
                builder.Append("--roll-forward");
                builder.Append(settings.RollForward.ToString());
            }
```

Let's only emit the `--roll-forward` if the user specifies a value for it, otherwise let `dotnet` use its default.